### PR TITLE
Implement CREATE PLACEMENT statement

### LINKED ISSUE #165: Support CREATE PLACEMENT statement
https://cloud.google.com/spanner/docs/reference/standard-sql/data-definition-language#create-placement

```
CREATE PLACEMENT `placement_name` [ partition_def ]

where partition_def is:
    { OPTIONS ( instance_partition="partition_id" [, default_leader="leader_region_id" ] ) }
```

### PR DESCRIPTION
**Pull request by apstndb**
This PR implements `CREATE PLACEMENT` DDL statement.

resolve #165

### CODE REVIEW

File: parser.go [hunk 1]
```diff
@@ -2421,6 +2424,18 @@ func (p *Parser) parseAlterDatabase(pos token.Pos) *ast.AlterDatabase {
 	}
 }
 
+func (p *Parser) parseCreatePlacement(pos token.Pos) *ast.CreatePlacement {
+	p.expectKeywordLike("PLACEMENT")
+	name := p.parseIdent()
```
Comment (makenowjust) [inline parser.go:2429]: According to the official doc, the `name` is enclosed in the backticks. Do we need to address this?a

Comment (apstndb) [reply parser.go:2429]: I have not tested in the real multi regional Enterprise Plus instance, but I believe the product accepts name without backquotes.

https://medium.com/google-cloud/distributed-ai-global-and-geo-partitioned-rag-architecture-with-google-cloud-spanner-for-aba9a75fc4c3

```
$ gcloud spanner databases ddl update news-international-database 
--instance=news-international-instance 
--ddl="CREATE PLACEMENT americas 
       OPTIONS (instance_partition='americas-partition')"
```

Comment (makenowjust) [reply parser.go:2429]: I see. Another question: there is no possibility of accepting `Path` as a name?

Comment (apstndb) [reply parser.go:2429]: It is a global object and all `PLACEMENT KEY` in all schemas of the database shares the same namespace.
https://cloud.google.com/spanner/docs/information-schema#placements

For example, `INFORMATION_SCHEMA.PLACEMENTS` doesn't have a room to have schema.

Column name | Type | Description
-- | -- | --
PLACEMENT_NAME | STRING | The name of the placement.
IS_DEFAULT | BOOL | A boolean that indicates whether the placement is the default placement.

Comment (makenowjust) [reply parser.go:2429]: Good. Thank you!

**Review by makenowjust — APPROVED** [review]

**Pull request merged**